src/components/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,25 @@
+import { CheckBoxLayout } from "./CheckBox.style";
+
+interface CheckBoxProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  dataTestId: string;
+}
+
+export function CheckBox({
+  checked = false,
+  id,
+  onChange,
+  dataTestId,
+  disabled = false,
+}: CheckBoxProps) {
+  return (
+    <input
+      type="checkbox"
+      css={CheckBoxLayout}
+      id={id}
+      checked={checked}
+      onChange={onChange}
+      data-testid={dataTestId}
+      disabled={disabled}
+    />
```

Comment:
```suggestion
export function CheckBox({
  ...props
}: React.InputHTMLAttributes<HTMLInputElement>) {
  return (
    <input
      type="checkbox"
      css={CheckBoxLayout}
      {...props}
    />
  );
}
```

이렇게 하면 훨씬 간결하게 표현할 수 있을 것 같은데, 캉골이 이전에 이렇게 스프레드 연산자로 사용하는 것을 좋아하지 않는다(?)라고 했던 것 같아요. 이렇게 `React.InputHTMLAttributes<HTMLInputElement>` 타입을 사용하는데 필요한 값만 들고와서 사용하는 것에 대한 장점이 무엇인지 궁금합니다!

저는 이렇게 스프레드 연산자를 사용하면 다음과 같은 장점이 있다고 생각합니다.

1. 코드가 간결하고 유지보수가 쉬움
별도로 id, checked, onChange, disabled 등을 하나씩 꺼낼 필요가 없음.
input 속성이 많아져도 컴포넌트가 더러워지지 않음.

2. 유연성 확보
사용자가 name, aria-*, onClick, value, required 등 어떤 속성이든 자유롭게 전달 가능.
특히 재사용성이 높은 디자인 시스템 컴포넌트에 적합.